tests/integration/test_vulnerability_detector/test_providers/test_providers_multiple_feeds_provider.py

```diff
@@ -0,0 +1,230 @@
+from wazuh_testing.modules import vulnerability_detector as vd
+
+# Marks
+pytestmark = [pytest.mark.server]
+
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+CUSTOM_FEED_PATH = os.path.join(os.path.dirname(os.getcwd()), 'integration', 'test_vulnerability_detector', 'data')
+
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'providers_multiple_feeds_provider.yaml')
+test_multiple_providers_cases_path = os.path.join(TEST_CASES_PATH, 'test_multiple_feeds_provider.yaml')
+custom_redhat_oval_feed_path = os.path.join(CUSTOM_FEED_PATH, 'feeds', vd.CUSTOM_REDHAT_OVAL_FEED)
+custom_redhat_json_feed_path = os.path.join(CUSTOM_FEED_PATH, 'feeds', vd.CUSTOM_REDHAT_JSON_FEED)
+custom_debian_oval_feed_path = os.path.join(CUSTOM_FEED_PATH, 'feeds', vd.CUSTOM_DEBIAN_OVAL_FEED)
+custom_debian_json_feed_path = os.path.join(CUSTOM_FEED_PATH, 'feeds', vd.CUSTOM_DEBIAN_JSON_FEED)
+configuration_parameters, configuration_metadata, test_case_ids = configuration.get_test_cases_data(
+    test_multiple_providers_cases_path)
+configurations = configuration.load_configuration_template(configurations_path, configuration_parameters,
+                                                           configuration_metadata)
+
+to_modify = ['CUSTOM_REDHAT_OVAL_FEED_PATH', 'CUSTOM_REDHAT_JSON_FEED_PATH', 'CUSTOM_DEBIAN_OVAL_FEED_PATH',
+             'CUSTOM_DEBIAN_JSON_FEED_PATH']
+new_values = [custom_redhat_oval_feed_path, custom_redhat_json_feed_path, custom_debian_oval_feed_path,
+              custom_debian_json_feed_path]
+configurations = configuration.update_configuration_template(configurations, to_modify, new_values)
+configuration_metadata = configuration.update_configuration_template(configuration_metadata, to_modify, new_values)
+
+
+def check_log_events(metadata):
+    """Check events in the ossec.log file.
+
+    Args:
+        metadata (dict): Metadata from the template.
+
+    Raises:
+        ValueError: If the provider in not in the test case dict.
+    """
+    evm.check_fetching_feed_log(metadata['oval_feed_path'])
+
+    if 'Debian' in metadata['provider_name']:
+        evm.check_fetching_feed_log(metadata['json_feed_path'])
+        evm.check_refresh_feed_log(metadata['provider_name'])
+        evm.check_provider_database_update_finish_log(metadata['provider_name'])
+    elif 'Red Hat' in metadata['provider_name']:
+        evm.check_refresh_feed_log(metadata['provider_name'])
+        evm.check_provider_database_update_finish_log(metadata['provider_name'])
+        evm.check_fetching_feed_log(metadata['json_feed_path'])
+        evm.check_refresh_feed_log(metadata['provider_json_name'])
+        evm.check_provider_database_update_finish_log(metadata['provider_json_name'])
+    else:
+        raise ValueError(f"No provider in the test case metadata list")
```

Comment:
```suggestion
        raise ValueError(f"Unexpected {metadata['provider_name']} provider in the test case metadata list")
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2538/commits/762b15217466b07a5e4bfa23bcf2c647e1848760